tcms/rpc/api/forms/testrun.py

```diff
@@ -21,40 +23,51 @@ def assign_plan(self, plan_id):
         self.populate(self.fields['plan'].queryset.first().product_id)
 
 
-class UpdateForm(NewForm):
-    plan = forms.ModelChoiceField(
-        queryset=TestPlan.objects.all(),
-        required=False,
-    )
-    summary = forms.CharField(
-        required=False
-    )
-    manager = forms.ModelChoiceField(
-        queryset=User.objects.all(), required=False
-    )
-    product = forms.ModelChoiceField(
-        queryset=Product.objects.all(),
-        empty_label=None, required=False
-    )
-    product_version = forms.ModelChoiceField(
-        queryset=Version.objects.none(),
-        empty_label=None, required=False
-    )
-    build = forms.ModelChoiceField(
-        queryset=Build.objects.all(),
-        required=False
+class UpdateForm(forms.ModelForm):
+
+    manager = UserField()
+    default_tester = UserField(required=False)
+
+    start_date = forms.DateTimeField(
+        input_formats=['%Y-%m-%d', '%Y-%m-%d %H:%M:%S'],
+        error_messages={
+            'invalid': _('The stop date is invalid. The valid format is YYYY-MM-DD [HH:MM:SS].')
```

Comment:
This is the `start_date`, but let's make the 2 strings the same so there is less to translate. Use something like 
"Invalid datetime format. Expected YYYY-MM ...."